app/views/bookings/_form.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= simple_form_for [@user, @boat, @booking] do |f| %>
```

Comment:
I believe you cannot have three objects in the array, only two. maybe we check with a TA